tracer/test/test-applications/instrumentation/CallTargetNativeTest/Derived.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+
+namespace CallTargetNativeTest
+{
+    internal abstract class AbstractClass
+    {
+        public abstract void VoidMethod(string name);
+    }
+
+    internal class Impl01OfAbstract : AbstractClass
+    {
+        public override void VoidMethod(string name)
+        {
+        }
+
+        public void OtherMethod()
+        {
+            Console.WriteLine("Hello from the other method");
+        }
+    }
+
+    internal class Impl02OfAbstract : AbstractClass
+    {
+        public override void VoidMethod(string name)
+        {
+        }
+    }
+
+    internal class NormalClassToDerive
```

Comment:
I don't see this class ever being called, was that intentional?

Reply:
Oh you're right, I think we need to remove it, there's no reference at all to this class, I guess was there initially and got replaced by NonAbstractClass.